test/serializer/abstract/ObjectAssign2.js

```diff
@@ -15,6 +15,7 @@ let dims = {
   }
 };
 
+if (global.__makeSimple) __makeSimple(dims);
```

Comment:
Can this break some case in RN that doesn't specify this?

Reply:
Yes. Let's find out.

Reply:
Although I'm actually curious if this is actually used because if it is, it might not be assigning any properties to it so it would be broken anyway.